src/generate-migration.ts

```diff
@@ -0,0 +1,36 @@
+import path = require("path");
+import * as fs from "fs";
+
+require('dotenv').config();
```

Comment:
dotenv is just a dev dependency and is used only in tests. here it should be removed. ENV should be set in the package/app where this package is dependency.